lib/content-services/src/lib/search/components/search-date-range-advanced-tabbed/search-date-range-advanced/search-date-range-advanced.component.spec.ts

```diff
@@ -0,0 +1,246 @@
+        fixture.detectChanges();
+        await fixture.whenStable();
+        await enterValueInInputField('date-range-advanced-between-start-input', '6/5/2023');
+        await enterValueInInputField('date-range-advanced-between-end-input', '6/10/2023');
+        expect(component.updatedDisplayValue.emit).toHaveBeenCalledWith('05-Jun-23 - 10-Jun-23');
+    });
+
+    it('should not update display label if anytime option is selected', async () => {
+        component.form.controls.dateRangeType.setValue(component.DateRangeType.ANY);
+        fixture.detectChanges();
+        await fixture.whenStable();
+        fixture.detectChanges();
+        expect(component.updatedDisplayValue.emit).toHaveBeenCalledWith('');
+    });
+
+    it('should not update display label if no valid value was provided', async () => {
+        component.form.controls.dateRangeType.setValue(component.DateRangeType.IN_LAST);
+        fixture.detectChanges();
+        await fixture.whenStable();
+        await selectDropdownOption('date-range-advanced-in-last-option-weeks');
+        fixture.detectChanges();
+        expect(component.updatedDisplayValue.emit).toHaveBeenCalledWith('');
+
+        component.form.controls.dateRangeType.setValue(component.DateRangeType.BETWEEN);
+        fixture.detectChanges();
+        await fixture.whenStable();
+        fixture.detectChanges();
+        expect(component.updatedDisplayValue.emit).toHaveBeenCalledWith('');
+    });
+
+    it('should set values if initial value is provided', () => {
+        let value: any = {
+            dateRangeType: component.DateRangeType.ANY
+        };
+        component.initialValue = value;
+        component.ngOnInit();
+        expect(component.form.controls.dateRangeType.value).toEqual(component.DateRangeType.ANY);
+
+        value = {
+            dateRangeType: component.DateRangeType.IN_LAST,
+            inLastValueType: component.InLastDateType.WEEKS,
+            inLastValue: '5'
+        };
+        component.initialValue = value;
+        component.ngOnInit();
+        expect(component.form.controls.dateRangeType.value).toEqual(component.DateRangeType.IN_LAST);
+        expect(component.form.controls.inLastValueType.value).toEqual(component.InLastDateType.WEEKS);
+        expect(component.form.controls.inLastValue.value).toEqual('5');
+
+        value = {
+            dateRangeType: component.DateRangeType.BETWEEN,
+            betweenStartDate: startOfYesterday(),
+            betweenEndDate: endOfToday()
+        };
+        component.initialValue = value;
+        component.ngOnInit();
+        expect(component.form.controls.dateRangeType.value).toEqual(component.DateRangeType.BETWEEN);
+        expect(component.form.controls.betweenStartDate.value).toEqual(startOfYesterday());
+        expect(component.form.controls.betweenEndDate.value).toEqual(endOfToday());
+    });
+    it('should not have any validators on any input fields when anytime option is selected', async () => {
```

Comment:
Please add empty line before that test

Reply:
Added empty line to make code more readable